### DISCUSSION TITLE
[dropdown] allowAdditions callback without multiple selection

### DISCUSSION THREAD

**Discussion by ivangomes** [fomantic/Fomantic-UI#2306] [Modules (Modal, Dropdown, Popup, Calendar, Toast, ...)] [answered]
Basically my dropdown will do a search and let me add a new label, but I am wondering how I can get a callback form the new label or value and insert it into my database.

I tried the callback onLabelCreate(value, text). But seems like it will only return if my dropdown allow multiple selections.

class="ui fluid search selection dropdown" 
 allowAdditions: true

Any ideas?

Thank you guys!

**Comment by lubber-de** [comment, marked as answer]
Use `onChange` and check for class `addition`
```javascript
$('.ui.dropdown').dropdown({
  allowAdditions: true,
  onChange: function(value, text, $item) {
    if ($item.hasClass('addition')) {
      console.log('added ' + value);
    }
  }
});
```

**Reply by ivangomes** [reply]
You are the best!  This will do it! Thank you very much!
